src/app/space/create/codebases/codebases-toolbar/codebases-toolbar.component.html

```diff
@@ -5,9 +5,12 @@
       (onFilterChange)="filterChange($event)"
       (onSortChange)="sortChange($event)">
     <ng-template #addCodebaseTemplate>
-      <div class="with-cursor-pointer" placement="left" tooltip="Add A Codebase"
+      <div class="with-cursor-pointer" placement="left" tooltip="Add a Codebase"
```

Comment:
Might want to try the bottom? When the window gets small, the tooltip is pushed off the page.

![screen shot 2017-11-20 at 3 55 40 pm](https://user-images.githubusercontent.com/17481322/33041245-a965018c-ce0b-11e7-92c8-b56a88fa25ee.png)

Reply:
I was unable to fix this, when it is on the left. When set to 'bottom', it causes this:
<img width="270" alt="screen shot 2017-11-20 at 5 35 21 pm" src="https://user-images.githubusercontent.com/4032718/33045005-3d606ed2-ce19-11e7-883c-5d6a8fa0a242.png">
<img width="673" alt="screen shot 2017-11-20 at 5 35 28 pm" src="https://user-images.githubusercontent.com/4032718/33045007-3efab77a-ce19-11e7-8910-df5042631398.png">

When set to "top", it causes a crazy jumping screen, as well as this:
<img width="695" alt="screen shot 2017-11-20 at 5 36 31 pm" src="https://user-images.githubusercontent.com/4032718/33045032-63b36fda-ce19-11e7-9d61-0d438e504c2d.png">

I'll try some CSS positioning changes, but the piece that is in place isn't super flexible.